instrumentation/khttp-0.1/src/test/groovy/KHttpClientTest.groovy

```diff
@@ -22,7 +22,7 @@ class KHttpClientTest extends HttpClientTest {
   @Override
   int doRequest(String method, URI uri, Map<String, String> headers, Closure callback) {
     // khttp applies the same timeout for both connect and read
-    def timeoutSeconds = Math.max(CONNECT_TIMEOUT_MS, READ_TIMEOUT_MS) / 1000
+    def timeoutSeconds = CONNECT_TIMEOUT_MS / 1000
```

Comment:
i think this probably fine, even though `khttp` uses this for both connect and read timeouts

let's keep and if it causes issues we can bump

Reply:
@trask should I set  `READ_TIMEOUT_MS` only for `KHttpClientTest`? I mean define `READ_TIMEOUT_MS` only in `KHttpClientTest` ?

Reply:
yeah, that's probably wise, and self-documenting, since khttp is a bit unusual